gnippy/config.py

```diff
@@ -40,7 +40,7 @@ def get_config(config_file_path=None):
         Returns a dictionary with all the possible configuration options,
         with None for the options that were not provided.
     """
-    if not os.path.isfile(config_file_path):
+    if os.path.isfile(config_file_path):
```

Comment:
Whoops. sorry was taking a different approach originally and must have forgotten to revert this part